runtime/compiler/p/codegen/CallSnippet.cpp

```diff
@@ -719,7 +719,7 @@ uint8_t *TR::PPCInterfaceCallSnippet::emitSnippetBody()
          {
          int32_t  *patchAddr = (int32_t *)getLowerInstruction()->getBinaryEncoding();
          intptrj_t addrValue = (intptrj_t)cursor;
-         if (!comp->compileRelocatableCode())
+         if (!comp->compileRelocatableCode() && !comp->isOutOfProcessCompilation())
```

Comment:
Can you add a comment on the `else` path here that explains to the reader that the immediate fields of the instructions being patched need to be clear for AOT and JITServer because the relocation `OR`s the relocated address into the fields and therefore can't tolerate any unexpectedly set bits? It will save someone a lot of grief in the future if they ever have to visit this part of the code.

Reply:
Done: dbd29ed